rpc/rpc.go

```diff
@@ -40,9 +40,10 @@ import (
 )
 
 const (
-	replyPostfix   = "-reply-to-"
-	statusCodeKey  = "status-code"
-	descriptionKey = "status-description"
+	replyPostfix           = "-reply-to-"
+	statusCodeKey          = "status-code"
+	descriptionKey         = "status-description"
+	defaultReceiverCredits = 1000
```

Comment:
Is this value similar to what other SDKs use?

Reply:
Servicebus management link (server side peer uses 5000

Reply:
Before the rpc had 1 link per call, so 1 made sense.
Now we share the link across many calls. You can’t use 1

Reply:
It doesn’t really matter what’s the value, unless you reach that kind of rpc call concurrency.

Reply:
Ah ok yes that makes sense.

Reply:
var Receiver = function (session, name, local_handle, opts) {
    this.init(session, name, local_handle, opts, true);
    this.drain = false;
    this.set_credit_window(this.get_option('credit_window', 1000));
    
    in the JS framework

Reply:
Yes, rhea (the library we use in JS) uses the same limit.